certbot/tests/client_test.py

```diff
@@ -201,7 +201,8 @@ def test_obtain_certificate(self, mock_crypto_util):
 
         authzr = []
 
-        for domain in domains:
+        # domain ordering should not be affected by authorization order
+        for domain in reversed(domains):
```

Comment:
Reversing the domains is meant to check that the order of the authorizations doesn't affect the subsequent order of the domains in the CSR.